docs/src/pages/components/data-grid/editing/editing.md

```diff
@@ -142,6 +150,14 @@ To enable it, change the edit mode to `"row"` using the `editMode` prop, then se
 
 {{"demo": "pages/components/data-grid/editing/BasicRowEditingGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
+### Service-side persistence [<span class="pro"></span>](https://mui.com/store/items/material-ui-pro/)
+
+If you want to send the edited data to your server, you can use the `onRowEditCommit` which is fired just before committing the new row value to the grid.
```

Comment:
```suggestion
If you want to send the updated data to the server, you can use the `onRowEditCommit` prop which is fired just before committing the new row value to the grid.
To access the new values for the row, use `apiRef.current.getEditRowsModel` to get all rows in edit mode, then use the id provided to get only the values for the row that was committed.
```

Reply:
If I stop using the `apiRef` in the demo, then we should probably add a "If you are using the pro plan" here.